project/project.godot

```diff
@@ -209,7 +209,6 @@ pointing/emulate_touch_from_mouse=true
 
 [rendering]
 
-quality/driver/driver_name="GLES2"
```

Comment:
Niech zostanie GLES2 domyślnie, jak ogarniemy particle to zrobimy GLES3 domyślne